src/algos/time_iteration.jl

```diff
@@ -103,11 +103,11 @@ If the stochastic process for the model is not explicitly provided, the process
 * `dr`: Solved decision rule.
 """
 function time_iteration(model, process, init_dr;
-      verbose::Bool=true, maxit::Int=100, tol::Float64=1e-8, infos::Bool=false)
+      verbose::Bool=true, maxit::Int=100, tol::Float64=1e-8, details::Bool=false)
 
     # get grid for endogenous
-    gg = model.options.grid
-    grid = CartesianGrid(gg.a, gg.b, gg.orders)  # temporary compatibility
+    grid = model.grid

```

Comment:
Yes, that's what we do on the python side. There is a `model.get_grid(**additional_args_which_override_the_content_from_yaml)` .
I guess it would be enough to extend the current `get_*` method (doesn't need to be redef. for Model) but I just want to know how to do it systematically for nested structures (for instance, if you want to change the domain while getting the grid).